react-components/src/hooks/pointClouds/usePointCloudAnnotationMappingForAssetId.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { type UseQueryResult, useQuery } from '@tanstack/react-query';
+import { type PointCloudAnnotationMappedAssetData } from '../types';
+import { EMPTY_ARRAY } from '../../utilities/constants';
+import { type AnyIntersection } from '@cognite/reveal';
+import { queryKeys } from '../../utilities/queryKeys';
+import { usePointCloudAnnotationCache } from '../../components/CacheProvider/CacheProvider';
+import { fetchAnnotationsForModel } from './fetchAnnotationsForModel';
+
+export const usePointCloudAnnotationMappingForAssetId = (
+  intersection: AnyIntersection | undefined
+): UseQueryResult<PointCloudAnnotationMappedAssetData[]> => {
```

Comment:
I was wondering if it is not better to use the hook name based on the input type and not based on what the useQuery uses as a queryKey input. 
for instance: `usePointCloudAnnotationMappingByIntersection`

Reply:
Nice catch :eyes: